src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -150,6 +150,30 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                         </button>
                     </span>
                 </div>
+                <div *ngIf="isAdmin">
```

Comment:
```suggestion
                <!--
                At the moment, this is only visible to admins as the actual main functionality has not yet been added on the server.
                In the future, this feature will be available to editors and instructors as well.
                -->
                <div *ngIf="isAdmin">
```
Adding some kind of explanation in the code why this is only available to admins should be helpful to anyone touching the code somewhen between this PR and the one adding the functionality.